vnext/Microsoft.ReactNative.Cxx/UI.Composition.h

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#ifdef USE_WINUI3
+#include <winrt/Microsoft.UI.Composition.h>
+#else
+#include <winrt/Windows.UI.Composition.h>
+#endif //  WINUI3
```

Comment:
Hmm... Clang format didn't pick up the changed definition. Will fix.

Reply:
Do you plan to send another iteration to fix comments?
Clang format updates comments for namespaces, but not for #ifdef.

---
In reply to: [424354647](https://github.com/microsoft/react-native-windows/pull/4886#discussion_r424354647) [](ancestors = 424354647)

Reply:
Yep. Will send out another one. VS added these but I'm not sure what it used to do that but not update :/